src/containers/CollectionPage.js

```diff
@@ -37,10 +39,17 @@ class CollectionPage extends React.Component {
   };
 
   render() {
-    const { collections, collection, publicFolder, page, entries } = this.props;
+    const { collections, collection, publicFolder, page, entries, isFetching } = this.props;
     if (collections == null) {
       return <h1>No collections defined in your config.yml</h1>;
     }
+
+    const noEntriesContent = isFetching
```

Comment:
We can clean this up and improve semantics by defining `loadingContent` and `noEntriesContent` separately, and saving the ternary for the return value.